src/h5web/providers/hdf5-models.ts

```diff
@@ -51,24 +51,25 @@ export type HDF5Type =
   | HDF5StringType
   | HDF5BooleanType
   | HDF5ComplexType
-  | HDF5Id
   | HDF5ArrayType
   | HDF5VLenType
   | HDF5CompoundType
-  | HDF5EnumType;
+  | HDF5EnumType
+  | HDF5UnknownType;
 
 export type HDF5NumericType = HDF5IntegerType | HDF5FloatType;
 
 export enum HDF5TypeClass {
-  Integer = 'H5T_INTEGER',
-  Float = 'H5T_FLOAT',
-  String = 'H5T_STRING',
-  Array = 'H5T_ARRAY',
-  VLen = 'H5T_VLEN',
-  Compound = 'H5T_COMPOUND',
-  Enum = 'H5T_ENUM',
-  Bool = 'H5T_BOOL',
-  Complex = 'H5T_COMPLEX',
+  Bool = 'boolean',
+  Integer = 'integer',
+  Float = 'float',
+  Complex = 'complex',
+  String = 'string',
+  Compound = 'compound',
+  Array = 'array',
+  VLen = 'vlen',
+  Enum = 'enum',
+  Unknown = 'unknown',
```

Comment:
New `Unknown` class. This will allow every type to inherit the same `{ class: HDF5TypeClass }` interface in my next PR.